src/utils.js

```diff
@@ -178,7 +182,9 @@ function getFailedHooks(suite) {
  */
 function cleanSuite(suite, totalTestsRegistered) {
   suite.uuid = uuid.v4();
-  const failedHooks = _.map(getFailedHooks(suite), cleanTest);
+  const hooks = getFailedHooks(suite);
+  const beforeFailedHooks = _.map(hooks.before, cleanTest);
+  const afterFailedHooks = _.map(hooks.after, cleanTest);
```

Comment:
I'd like to see this just be
```js
const beforeHooks = _.map([].concat(suite._beforeAll, suite._beforeEach), cleanTest);
const afterHooks = _.map([].concat(suite._afterAll, suite._afterEach), cleanTest);
```
It's simpler and the renderer can handle which hooks are shown.